### DISCUSSION TITLE
cors configuration dose not work

### DISCUSSION THREAD

**Discussion by bouyeijiang** [actix/actix-web#2865] [Q&A]
dependencies:
```
actix-web="4"
actix-files="0.6.0"
actix-cors="0.6.2" 
```

```rust
HttpServer::new(|| {
         let cors= Cors::default()
        .allow_any_origin()
        .send_wildcard()
         .allow_any_method()
         .allow_any_header();
        App::new()
        .wrap(cors)
    })
    .bind(("127.0.0.1",8088))?
    .run()
    .await
```

//browser result:
CORS header missing  'Access-Control-Allow-Origin'

**Comment by robjtede** [comment]
Make sure you understand the [CORS protocol](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) and try using `Cors::permissive()` to get started with actix-cors.

**Comment by bouyeijiang** [comment]
it's useless.I don't know if there are other settings
